examples/guides/se-restful-webservice/README.adoc

```diff
@@ -250,208 +200,180 @@ import io.helidon.webserver.ServerRequest;
 import io.helidon.webserver.ServerResponse;
 import io.helidon.webserver.Service;
 ----
-
-. Make `GreetService` implement `io.helidon.webserver.Service`.
-. Set two static fields by reading the config file and setting the initial greeting from
-the loaded config:
+for JSON and config support and for the key parts of the web server.
+--
+. The `GreetService` class implements `io.helidon.webserver.Service`.
+. Its constructor accepts a `Config` object to control its behavior:
 +
 --
 [source,java]
-// _include::0-1:{greet-service}[tags=CONFIG;greetingDef]
+// _include::0-2:{greet-service}[tag=ctor,indent=0]
 ----
-    private static final Config CONFIG = Config.create().get("app"); // <1>
-    private String greeting = CONFIG.get("greeting").asString().orElse("Ciao"); // <2>
+GreetService(Config config) {
+    this.greeting = config.get("app.greeting").asString().orElse("Ciao"); //<1>
+}
 ----
-<1> Loads the config from the (default) `application.yaml` resource you created earlier
-and loads the subtree rooted at the `app` entry into the `CONFIG` field. The
-type of the field is `io.helidon.config.Config`.
-<2> Sets the initial greeting from the config, using "Ciao" if the expected
-entry is missing from the loaded config.
+Here the code looks up the initial greeting from the configuration object
+and saves it.
 --
-. Implement the responses to the three types of request by adding three methods.
-.. Returning the default greeting
+. The `update` method updates the routing rules in the web server, thereby
+linking the service's methods with the corresponding URL paths:
 +
 --
 [source,java]
-// _include::0-8:{greet-service}[tag=getDefaultMessage]
+// _include::0-6:{greet-service}[tags=update;!updateForCounter,indent=0]
+----
+@Override
+public void update(Routing.Rules rules) {
+    rules
+        .get("/", this::getDefaultMessageHandler) //<1>
+        .get("/{name}", this::getMessageHandler) //<2>
+        .put("/greeting/{greeting}", this::updateGreetingHandler); //<3>
+}
+----
+<1> Handle `GET` requests with no extra path using `getDefaultMessage`.
```

Comment:
I think "with" here means "that contain". 

If yes, consider changing to: "Handle `GET` requests that contain no extra path ...".

Reply:
Done